stack.yaml.lock

```diff
@@ -0,0 +1,47 @@
+# This file was autogenerated by Stack.
```

Comment:
We should probably make it a conscious decision whether to check in this lockfile (in a seperate PR), rather than it happening "by accident".

Reply:
I'd prefer to add it to gitignore honestly. I'm pretty sure we've never come up against a situation where it would have been useful to have.

Reply:
Sorry, didn't mean to commit this in the first place :) I've removed it